docs/code/mecsimcalc-library.md

```diff
@@ -0,0 +1,155 @@
+---
+sidebar_label: "mecsimcalc Library"
+sidebar_position: 4
+---
+
+# Mecsimcalc 0.0.2 documentation
+
+This library is designed to provide a set of functions for handling and converting various types of data, such as base64 encoded data, Pandas DataFrames, and Pillow images.
+
+```python
+from PIL import Image
+import base64
+import io
+import pandas as pd
+```
+
+## General
+
+### decode_file_data
+
```

Comment:
For each of the functions, can you add a line either at the top or bottom that says `[source code]` and hyperlink to where that function is the in the source code. Use the github line link where you can link to a specific line

Reply:
Can you do this for all the functions